app/controllers/application_controller.rb

```diff
@@ -9,6 +9,11 @@ def logged_in?
   end
   helper_method :logged_in?
 
+  def am_i_admin?
+    current_user.present? and current_user.admin
+  end
+  helper_method :am_i_admin?
+
```

Comment:
Might be useful for later in view logics. (I've used it) If there's cleverer solution, let me know! :pray: